Rules/AvoidUsingInternalURLs.cs

```diff
@@ -118,7 +113,12 @@ public IEnumerable<DiagnosticRecord> AnalyzeScript(Ast ast, string fileName)
                             }
                             if (!firstPartURL.Contains("."))
                             {
-                                isInternalURL = true;
+                                //Add a check to exclude potential SDDL format. Check if a string have four components separated by ":"
+                                var count = firstPartURL.Count(x => x == ':');
+                                if (count !=3 )
+                                {
+                                    isInternalURL = true;
+                                }
```

Comment:
I think an SDDL can actually have 4 ":"
This is because they have DACL and SACL components.
https://msdn.microsoft.com/en-us/library/windows/desktop/aa379570(v=vs.85).aspx